kythe-verification/testdata/basic/ImportsRef.hs

```diff
@@ -0,0 +1,8 @@
+-- Verify the references to the imported modules
```

Comment:
Alternative idea: does Kythe verifier unify the node names across multiple source files when running on multi sources? Then could add a two-file test, and verify that file A exports ModuleA, and file B imports ModuleA (could make a negative test to find out if this works, with three files: B imports A, but verifier asserts it imports C - it should fail). Maybe.